src/meetings/components/meetingAccessPoint/AccessMeetingPage.test.tsx

```diff
@@ -400,7 +400,7 @@ describe('AccessMeeting - enter to meeting by waiting Room', () => {
 			});
 		});
 
-		expect(mockGoToInfoPage).toBeCalledWith(PAGE_INFO_TYPE.MEETING_ENDED);
+		expect(mockGoToInfoPage).toHaveBeenCalledWith(PAGE_INFO_TYPE.MEETING_ENDED);
```

Comment:
Since these two methods are the same, because toBeCalledWith is an alias of toHaveBeenCalledWith did you changed for keep as default method or for other reasons?

Reply:
I changed it because the toBeCalledWith is deprecated, as other aliases, in favour of the native ones

Reply:
BEry nice and ty since I missed this updated!